### ISSUE TITLE
PUP, SMN, BST pets will not summon in some dungeon zones

### ISSUE THREAD

**Issue by setsuk0** [EdenServer/community#2890]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details

Client version 30181205_0

Summoning automaton in Maze of Shakrami does not work. The ability goes on cooldown but automaton does not appear. We tried this twice with two different PUPs to confirm. To reproduce, enter the zone on PUP and use activate to summon automaton. 

I saw a player shouting earlier today that his BST jug pets could not be summoned in Shakrami either. I assume this is related.

**Comment by zuultek** [comment]
Had same issue in Sacrarium just now, both SMNs in pt stopped being able to summon avatars. Everything worked in the beginning, then suddenly stopped.

**Comment by setsuk0** [comment]
Seems like an issue with pets in "dungeons" so I'll update the title of the bug report. Thanks Zuultek.

**Comment by bkesecker** [comment]
The issue is now beyond dungeons and will likely spread to all zones where pets are able to be summoned.  We do have an idea why, but the quick fix would re-introduce a crash fix.

**Comment by bkesecker** [comment]
Expected fix is live.  Issue will auto-close once the fix is merged into the main branch.
